converter/value.go

```diff
@@ -48,4 +50,17 @@ type (
 		// json.Unmarshal.
 		Get(valuePtr ...interface{}) error
 	}
+
+	// RawValue is a representation of an unconverted, raw payload.
+	//
+	// This type can be used as a paramter or return type in workflows and activities to pass through
+	// a raw payload. Encoding/decoding of the payload is still done by the system.
+	RawValue struct {
+		Payload *commonpb.Payload
```

Comment:
You might consider not exporting this field and having a `Payload()` method instead. This discourages people from creating the struct themselves. The benefit of discouraging this is that it is less likely for people to use a `*RawValue` instead of a `RawValue` since the must use `NewRawValue` and see its result type is non-pointer.